File: play-services-base/src/main/java/com/google/android/gms/common/api/HasApiKey.java
/*
 * SPDX-FileCopyrightText: 2020, microG Project Team
 * SPDX-License-Identifier: Apache-2.0
 */

package com.google.android.gms.common.api;

import androidx.annotation.NonNull;
import com.google.android.gms.common.api.internal.ApiKey;
import org.microg.gms.common.Hide;

public interface HasApiKey<O extends Api.ApiOptions> {
    @NonNull
    @Hide
    ApiKey<O> getApiKey();
}

File: play-services-base/src/main/java/com/google/android/gms/common/api/HasApiKey.kt
/*
 * SPDX-FileCopyrightText: 2020, microG Project Team
 * SPDX-License-Identifier: Apache-2.0
 */
package com.google.android.gms.common.api

import com.google.android.gms.common.api.Api.ApiOptions
import com.google.android.gms.common.api.internal.ApiKey

interface HasApiKey<O : ApiOptions?> {
    fun getApiKey(): ApiKey<O>?
}